CRM/Admin/Form/PaymentProcessor.php

```diff
@@ -367,7 +367,7 @@ public function setDefaultValues() {
       $defaults['payment_processor_type_id'] = $this->_paymentProcessorType;
     }
     else {
-      CRM_Core_Session::setStatus('Payment Processor Type (ID=' . $this->_paymentProcessorType . ') not found. Did you disable the payment processor extension?', 'Missing Payment Processor', 'alert');
+      CRM_Core_Session::setStatus(ts('Payment Processor Type (ID=%1) not found. Did you disable the payment processor extension?', [1 => $this->_paymentProcessorType]), 'Missing Payment Processor', 'alert');
```

Comment:
You can also add `ts()` around the title "Missing Payment Processor"

Reply:
Good spot, I've pushed a change.